tracks/python/exercises/grains/mentoring.md

```diff
@@ -0,0 +1,67 @@
+# Reasonable solution by calculating power.
+
+```python
+def square(number):
+    """
+    This Function must return the number of grains on a given square number.
```

Comment:
"Function" is not a proper name, and should not be capitalized.  Same thing happens further on as well.